### ISSUE TITLE
[Идея] Синхронизация черного списка через заметки на авито

### ISSUE THREAD

**Issue by xob0t** [SyrnikovPavel/ave_blacklist#7]
У авито есть функционал заметок для объявлений.
Интрефейс для работы с ними есть только на `avito.ru`, но зато у них есть api.
Получить заметки
```
https://www.avito.ru/web/1/notes/list?itemIds=4222343220%2C2543064189
```
А создать или изменить вот так
```
https://www.avito.ru/web/1/notes/update
```
Тело запроса тут приводить не буду.
Есть ограничение символов, 180 на одну заметку, но для этих целей нам хватит.

Так вот, статус блокировки объявления можно записывать в заметку, и читать при загрузке страницы.
Таким образом, блеклист будет зависеть не от локальной базы, а от самого аккаунта.
Не знаю, насколько такой вариант будет удобен или жизнеспособен, но сама идея мне кажется интересной.
Ну и у авито нет заметок для продавцов, так что полноценно перенсети базу таким образом вряд-ли получится.

**Comment by SyrnikovPavel** [comment]
Да, идея классная). Но немного не соответствует глобальному направлению расширения. Скорее всего придется делать свой сервер для хранения и обработки данных для того: чтобы обеспечить сохранность данных (сейчас пользователи жалуются, что база периодически слетает и у них есть подозрения, что это авито нам платит, чтобы мы вредили таким образом 😆 ); не зависеть от работы сторонних сервисов (как например, заметки на авито - сегодня есть, завтра нет; или гугл завтра сделает какое-нибудь ограничение к хранилищу не совместимое с работой расширения и все); и самое главное - это реализация автоматической блокировки спамеров. Идея в том, чтобы автоматом блокировать пользователей, если они находятся в ЧС у n пользователей). Или как то выделять их. Короче говоря, на максимум использовать силу сообщества)
